python/lsst/afw/display/interface.py

```diff
@@ -644,6 +644,26 @@ def erase(self):
         """
         self._impl._erase()
 
+    def centroids(self, catalog, symbol=None, *args, **kwargs):
+        """Draw the sources from a catalog at their pixel centroid positions
+        as given by ``getX()`` and ``getY()``.
+
+        See `dot` for explanation of ``symb`` and available kwargs.
```

Comment:
It's spelled `symbol` in the function def.
Make it more explicit that `args, kwargs` are passed to `dot`.